conversions.go

```diff
@@ -206,10 +248,29 @@ func convertInMessage(
 		}
 
 		names := inMsg.ConsumeBytes(inMsg.Len())
+
 		// names should be "old\x00new\x00"
 		if len(names) < 4 {
 			return nil, errors.New("Corrupt OpRename")
 		}
+
+		// macFuse 4.2.1 workaround: the kernel implementation is buggy.
+		// It call OpRename with the data type of OpRename2. The workaround consists
+		// in skipping the first 8 bytes of names (which are zeroes).
+
```

Comment:
Remove empty line

Reply:
This workaround was probably already merged in my PR https://github.com/jacobsa/fuse/pull/107 %)